scripts/generate_version.py

```diff
@@ -0,0 +1,39 @@
+import json
+import subprocess
+import sys
+from time import strftime
+
+
+def getVersionInfo():
```

Comment:
can you use `get_version_info`, `show_usage`, etc. to be more consistent with Python's naming convention?